server/sockets/index.js

```diff
@@ -67,6 +94,9 @@ const socketListenTo = server => {
     updateItemWS(socket);
     addCommentWS(socket);
     cloneItemWS(socket);
+    setVoteWS(socket);
+    clearVoteWS(socket);
+    markAsDoneWS(socket, dashboardClients);
```

Comment:
1. alphabetic order
2. Does this method change item to unhandled also? If so, its name shouldn't be `markAsDoneWS` (changeOrderStateWS, toggleOrderStatus etc... )